website/docs/d/firewall.html.markdown

```diff
@@ -36,11 +36,11 @@ The following attributes are exported:
 
 * `id` - The Resource ID of the Azure Firewall.
 
-* `ip_configuration` - A `ip_configuration` block as defined below.
+* `ip_configurations` - A `ip_configurations` block as defined below.
 
```

Comment:
We will need to revert all references back to be `ip_configuration`, WDYT?

Reply:
Agree and reverted.